lua/telescope/command.lua

```diff
@@ -15,6 +48,13 @@ local bool_type = {
   ["true"] = true,
 }
 
+local split_keywords = {
+  ["find_command"] = true,
+  ["vimgrep_arguments"] = true,
+  ["sections"] = true,
+  ["search_dirs"] = true,
+}
```

Comment:
I think you should add `symbols` to this map. https://github.com/nvim-telescope/telescope.nvim/issues/901#issuecomment-891670369